PubHub/PubHub.API/Controllers/AuthController.cs

```diff
@@ -22,28 +22,34 @@ namespace PubHub.API.Controllers
     [Route("[controller]")]
     [Consumes(MediaTypeNames.Application.Json)]
     [ProducesResponseType(StatusCodes.Status400BadRequest, Type = typeof(ProblemDetails))]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized, Type = typeof(ProblemDetails))]
```

Comment:
I suppose this is because all endpoints have the potential for failed authentication when requsting data with an invalid app ID?